### PR TITLE
Bitcoincash payments

### PR DESCRIPTION
**Pull request by dwolfgram**
@dylanseago right now the only thing not passing is still the fee's not matching up for one of the tests. You will see i changed the min fee to 1 in the constants but not sure if i need to change anything else to account for bch's lower fees?

### CODE REVIEW

**Comment by dwolfgram** [comment]
also i ended up using https://github.com/junderw/bitcoinjs-lib/tree/cashv5. if you're ok with using it maybe we could make a fork or something?

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ See [@faast/coin-payments README](./packages/coin-payments/README.md) for usage.
 - [@faast/tron-payments](./packages/tron-payments)
 - [@faast/stellar-payments](./packages/stellar-payments)
 - [@faast/ripple-payments](./packages/ripple-payments)
+- [@faast/litecoin-payments](./packages/litecoin-payments)
```
Comment (dylanseago) [inline README.md:64]: Could you add bitcoincash and ethereum to this list too?

File: packages/bitcoincash-payments/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+bitcoincash-payments
```
Comment (dylanseago) [inline packages/bitcoincash-payments/README.md:1]: This documentation is way out of date, as are all the other readme's other than coin-payments. Could you just replace it with something that says refer to coin-payments README with a link to it?

File: packages/bitcoincash-payments/src/constants.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+export const DEFAULT_DERIVATION_PATHS = {
+  [AddressType.Legacy]: "m/44'/145'/0'",
+}
+
+export const DEFAULT_NETWORK = NetworkType.Mainnet
+
+export const NETWORK_MAINNET = networks.bitcoin
+export const NETWORK_TESTNET = networks.testnet
+
+export const DEFAULT_MAINNET_SERVER = process.env.BITCOIN_SERVER_URL
```
Comment (dylanseago) [inline packages/bitcoincash-payments/src/constants.ts:45]: These env var names need to be adjusted to BITCOIN_CASH

File: packages/bitcoincash-payments/test/fixtures/accounts.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    '2': 'bitcoincash:qr5avn7f6qs3m4497mf3zag5fkcuzke9qcl944t87w',
+    '5': 'bitcoincash:qpryvhmlqskajs4jzc77ja5gwfnxkpqf25acz3lz50',
+    '6': 'bitcoincash:qqm6lpfufywc42ufzhuxs4j5qy0dmcdmfu5r4567tw',
+    '10': 'bitcoincash:qps6errksj3an0hfzw09a6mdmtzw974klcs2cnghhm',
+    '10000': 'bitcoincash:qrx8fv76tp2x0c7fyh9whfxrwlp6ywvwesfkz3lvud',
+    '20000': 'bitcoincash:qqnwqxpj3nq5s20x4alem4g6y540w78tksf46694ej'
+  },
+}
+
+export const legacyFixedAccount: AccountFixture = {
```
Comment (dylanseago) [inline packages/bitcoincash-payments/test/fixtures/accounts.ts:34]: Is it necessary to have both of these fixtures?

File: packages/bitcoin-cash-payments/package.json [hunk 5]
```diff
@@ -0,0 +1,96 @@
+{
+  "name": "@faast/bitcoincash-payments",
```
Comment (dylanseago) [inline packages/bitcoin-cash-payments/package.json:2]: This name and other fields in package.json don't reflect the new `bitcoin-cash-payments` name

**Pull request merged**